etsin_finder/frontend/js/components/qvain/index.jsx

```diff
@@ -27,10 +27,19 @@ import handleSubmitToBackend from './utils/handleSubmit'
 import { getResponseError } from './utils/responseError'
 import Title from './general/title'
 import SubmitResponse from './general/submitResponse'
-import Button, { InvertedButton } from '../general/button';
+import { Button, InvertedButton } from '../general/button'
+import Modal from '../general/modal'
 import DeprecatedState from './deprecatedState';
 import PasState from './pasState'
 
+const customStyles = {
+  content: {
+    minWidth: '20vw',
+    maxWidth: '800px',
+    padding: '3em',
```

Comment:
It is confusing to use three different units of length in the same place.
```suggestion
    minWidth: '20vw',
    maxWidth: '800px',
    padding: '3em',
```